### PR TITLE
Enable NERDTree folder icons and autorefresh

### PR DESCRIPTION
**Pull request by scaramagus**
Continuation of PR #113

### CODE REVIEW

File: config.vim [hunk 1]
```diff
@@ -231,7 +231,7 @@ set wildmode=list:longest
 ca w!! w !sudo tee "%"
 
 " tab navigation mappings
-map tt :tabnew 
+map tt :tabnew
```
Comment (fisadev) [inline config.vim]: this space is meaningfull and needed, the mapping leaves the cursor at the end of the command, with a space in between, so the user can star typing the path if needed

File: config.vim [hunk 2]
```diff
@@ -344,7 +367,7 @@ nmap ,D :tab split<CR>:call jedi#goto()<CR>
 " Ack.vim ------------------------------
 
 " mappings
-nmap ,r :Ack 
+nmap ,r :Ack
```
Comment (fisadev) [inline config.vim]: this space is meaningfull and needed, the mapping leaves the cursor at the end of the command, with a space in between, so the user can star typing the search text

File: config.vim [hunk 3]
```diff
@@ -202,7 +202,7 @@ set shiftwidth=4
 set nu
 
 " remove ugly vertical lines on window division
-set fillchars+=vert:\ 
+set fillchars+=vert:\
```
Comment (fisadev) [inline config.vim]: this space is meaningful and needed, it's telling that the character to separate windows is a space

**Comment by scaramagus** [comment]
Editor at it again... only relevant changes added now

**Comment by fisadev** [comment]
Thank you!! And nice couple of features :)

**Pull request merged**